packages/block-editor/src/components/url-input/style.scss

```diff
@@ -124,6 +124,11 @@ $input-size: 300px;
 	box-shadow: $shadow-popover;
 	border: 1px solid $light-gray-500;
 	background: $white;
+
+	.components-base-control:last-child,
+	.components-base-control:last-child .components-base-control__field {
+		margin-bottom: 0;
+	}
```

Comment:
This might be necessary, but it didn't fix the styling problem for me. Adding the following rule, removed the extra space below the input:

```
.block-editor-url-input > .components-base-control__field {
	margin-bottom: 0;
}
```

@talldan is this the margin that you were referring to?

Reply:
Hey @pablinos. Yep, that looks like it.